Packs/CSCDomainManager/Integrations/CSCDomainManager/CSCDomainManager.py

```diff
@@ -0,0 +1,553 @@
+    """
+    Create a dict of the domain with the fields for human readable, using the domain argument
+
+    Args:
+        domain: domain dict that was output from the http request
+
+    Returns:
+        A dict of the domain with the fields for human readable
+    """
+    filtered = {'Qualified Domain Name': domain.get('qualifiedDomainName'),
+                'Domain': domain.get('domain'),
+                'Idn': domain.get('idn'),
+                'Generic top-level domains': domain.get('newGtld'),
+                'Managed Status': domain.get('managedStatus'),
+                'Registration Date': domain.get('registrationDate'),
+                'Registry Expiry Date': domain.get('registryExpiryDate'),
+                'Paid Through Date': domain.get('paidThroughDate'),
+                'Country Code': domain.get('countryCode'),
+                'Server Delete Prohibited': domain.get('countryCode'),
+                'Server Transfer Prohibited': domain.get('serverDeleteProhibited'),
+                'Server Update Prohibite d': domain.get('serverTransferProhibited'),
+                'Name Servers': domain.get('nameServers'),
+                'Dns Type': domain.get('dnsType'),
+                'Whois Contact first Name': domain.get('whoisContacts')[0].get('firstName'),
+                'Whois Contact last Name': domain.get('whoisContacts')[0].get('lastName'),
+                'Whois Contact email': domain.get('whoisContacts')[0].get('email')
+                }
+
+    domain.get('whoisContacts')
+    return filtered
+
+
+def get_whois_contacts_fields_for_search_domains(whois_contacts, field_name: str) -> list:
+    """
+    Create a list of contact.field_name for each contact in whois_contacts. Specific arrangement for the search domains command
+
+    Args:
+        whois_contacts: list of contacts
+        field_name: the field to get for each contact
+
+    Returns:
+        A list of contact.field_name
+    """
+    results = []
+    for contact in whois_contacts:
+        results.append(contact.get(field_name))
+
+    return results
+
+
+def get_whois_contacts_fields_for_domain(whois_contact, field_names: str | List[str],
+                                         contact_type_condition: str) -> list:
+    """
+    Create a list of contact.field_name for each contact in whois_contacts. Specific arrangement for the domain command
+
+    Args:
+        whois_contacts: list of contacts
+        field_name: the field to get for each contact
+        contact_type_condition: a str to use for condition: choose the contacts with the specific contact_type
+
+    Returns:
+        A list of contact.field_name when contact is from type contact_type_condition
+    """
+    results = []
+    if isinstance(field_names, str):
+        field_names = [field_names]
+
+    for contact in whois_contact:
+        if contact.get('contactType') == contact_type_condition:
+            combined_fields = ' '.join(contact[field] for field in field_names)
```

Comment:
we're concatenating all this to return in 1 field instead of an array? This seems very strange behavior, lets check what the standard is in other integrations

Reply:
usually we will send just one field in field_names but there is one option when we need to concatenate first + second name. i wanted to make this function general and to match both cases

Reply:
you mean this is a utility function to seperate fields by spaces? lets give this a better name then